keyboards/onepiece/qk21/config.h

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 linlin012 (@linlin012)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+#define FORCE_NKRO
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+// The pin connected to the data pin of the LEDs
+#define RGB_DI_PIN B7

```

Comment:
This is changing at the end of the month. 
```suggestion
#define RGB_DI_PIN B7
#define WS2812_DI_PIN B7
```
or just:
```
#define WS2812_DI_PIN B7
```
(will fail to compile until develop is merged at end of month)